lms/static/sass/base/_utilities.scss

```diff
@@ -0,0 +1,5 @@
+.sr-is-focusable,
```

Comment:
NIT: mind adding a PL-worthy Sass comment header to this file noting what its used for (I know its not the norm for these old nasty Sass partials, but we gotta start sometime, right)?